LineaPro.js

```diff
@@ -1,80 +1,79 @@
-'use strict'
```

Comment:
The format plugin Prettier got a hold of this file. If you'd like me to reverse it let me know!